deploy-queue/action.yml

```diff
@@ -172,3 +172,47 @@ runs:
             exit 1
             ;;
         esac
+
+    - name: Send start notification to Slack
+      id: send-slack-start
+      if: steps.run-deploy-queue.outcome == 'success' && inputs.mode == 'start' && inputs.environment == 'prod'
```

Comment:
I would probably remove the check for `inputs.environment == 'prod'`, if we want to send a message from dev as well